### ISSUE TITLE
Inline display does not allow using altField option

### ISSUE THREAD

**Issue by buttflattery** [yiisoft/yii2-jui#87]
### What steps will reproduce the problem?
Create a datepicker with the following options where you use `inline` option along with the `altField` option

```php
echo DatePicker::widget([
    'name' => 'date_range',
    'inline' => true,
    'clientOptions' => [
        'selectOtherMonths' => true,
        // 'inline' => true,
        'altField' => '#shootssearch-created_at',
    ]
    //'language' => 'ru',
    //'dateFormat' => 'yyyy-MM-dd',
]);
```
and create an active field using `yii\helpers\Html` or `ActiveForm` to be used as an actual form field to be submitted along with the datepicker value when form is submitted. 

```php
echo Html::activeTextInput($model, 'created_at');
```
### What's expected?
i expect the field to be populated with the date value selected from the datepicker 

### What do you get instead?
nothing happens and the field stays blank

The reason behind is that for `inline` display you have the following code on line 201 in `renderWidget()` function

```php
$this->clientOptions['altField'] = '#' . $this->options['id'];
```
which overrides the custom defined `altField` .

you can change that line to something like below which will work for both cases
- default hidden field created by the widget.
- the custom field if the user defines one using the `altField` .

```php
$customAltField = yii\helprs\ArrayHelper::getValue($this->clientOptions, 'altField', false);
$this->clientOptions['altField'] = '#' . $this->options['id'];
if ($customAltField) {
    $this->clientOptions['altField'] .=  ', ' . $customAltField;
}
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       | 2.0.20
| PHP version      | 5.6
| Operating system | Windows 10

**Comment by samdark** [comment]
Thanks for reporting. Feel free to do a pull request.

**Comment by buttflattery** [comment]
@samdark sure, but i havent contributed before in any of Yii's repo, are there any guidelines to follow ?

**Comment by samdark** [comment]
Yes: https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md

**Comment by buttflattery** [comment]
@samdark what i understand is that i have to fork and clone the main `Yii2` repo first to contribute the fix to this extension. am i right ? or should i do those 3 steps for the extensions repo directly rather than the Main `Yii2` repo first? which i dont think so is the right way.

Sorry to bug you about this.

**Comment by samdark** [comment]
Yes, you are right. Yii2 repo first.

**Comment by samdark** [comment]
Any luck with the pull request?

**Comment by buttflattery** [comment]
Been busy lately, just going to push the code can you tell if the `CHANGELOG` entry will go under `2.0.8 under development` section ? i am sure it goes under that.

**Comment by buttflattery** [comment]
@samdark just created the pull request , is this merging message normal or have i done something wrong 
![image](https://user-images.githubusercontent.com/1536454/61825902-b5273700-ae7a-11e9-979a-e7cdd221c7c2.png)

**Comment by samdark** [comment]
Merging is blocked as expected since we need to check the pull request first.

**Comment by buttflattery** [comment]
@samdark alright, and dont  you think a test for the `altField` with `inline` display mode should be added too.

**Comment by samdark** [comment]
Yes, sure it should.
